data/json/overmap_specials.json

```diff
@@ -220,6 +220,20 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Sugar House",
+        "overmaps" : [
+            { "point":[0,0,0], "overmap": "sugar_house" }
```

Comment:
Since `"sugar_house"` now rotates, it needs a direction here (e.g. `"sugar_house_north"`, depending on facing towards the road). Sorry for not mentioning it earlier.